src/chat/summarizers/topics.ts

```diff
@@ -0,0 +1,18 @@
+import { Subject } from "../../models/schema";
+import { KafkaTopic } from "../../models/topic";
+
+/** Create a Markdown-formatted list of {@link KafkaTopic} names and their children as {@link Subject}. */
+export function summarizeTopics(topic: KafkaTopic): string {
```

Comment:
```suggestion
export function summarizeTopic(topic: KafkaTopic): string {
```
Minor rename for accuracy